auction-server/src/auction/service/verification.rs

```diff
@@ -673,11 +783,16 @@ impl Verification<Svm> for Service<Svm> {
             transaction:        bid.chain_data.transaction.clone(),
         };
         let permission_key = bid_chain_data.get_permission_key();
+        let submit_type = self.get_submission_state(&permission_key).await;
```

Comment:
The simulation takes like 100ms and during this time, the state for the submit_type may change (It can become invalid). Leave a comment here to take care of this latency later.